config/hr_paragraphs.settings.yml

```diff
@@ -2,3 +2,4 @@ sync_domain: 'http://hrinfo.docksal.site'
 sync_credentials: ''
 csv_source_dir: /var/www/tsv/
 notification_new_user_enabled: false
+reliefweb_api_endpoint: 'https://api.reliefweb.int/v1/reports'
```

Comment:
Missing `reliefweb_api_appname`.

Also those 2 new properties need to be added to the schema: https://github.com/UN-OCHA/response-site/blob/develop/html/modules/custom/hr_paragraphs/config/schema/hr_paragraphs.schema.yml

And should be added there as well: https://github.com/UN-OCHA/response-site/blob/develop/html/modules/custom/hr_paragraphs/config/install/hr_paragraphs.settings.yml